packages/rum-core/src/browser/htmlDomUtils.spec.ts

```diff
@@ -59,21 +59,42 @@ describe('isElementNode', () => {
 
 if (!isIE()) {
   describe('isShadowRoot', () => {
-    const parent = document.createElement('div')
-    parent.attachShadow({ mode: 'open' })
-    const parameters: Array<[Node, boolean]> = [
-      [parent.shadowRoot!, true],
-      [parent, false],
-      [document.body, false],
-      [document.createTextNode('hello'), false],
-      [document.createComment('hello'), false],
+    const notShadowDomNodes: Node[] = [
+      document,
+      document.head,
+      document.body,
+      document.createElement('div'),
+      document.createTextNode('hello'),
+      document.createComment('hello'),
     ]
 
-    parameters.forEach(([element, result]) => {
-      it(`should return ${String(result)} for "${String(element)}"`, () => {
-        expect(isNodeShadowRoot(element)).toBe(result)
+    notShadowDomNodes.forEach((element) => {
+      it(`should return false for "${String(element.nodeName)}"`, () => {
+        expect(isNodeShadowRoot(element)).toBe(false)
       })
     })
+
+    it('should return true for shadow root but not its host', () => {
+      const parent = document.createElement('div')
+      const shadowRoot = parent.attachShadow({ mode: 'open' })
+      expect(isNodeShadowRoot(parent)).toBe(false)
+      expect(isNodeShadowRoot(shadowRoot)).toBe(true)
+    })
+
+    it('should return false for a[href] since it has a host property', () => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** I'd say that it returns false since it is not a document fragment
‏
```suggestion
    it('should return false for a[href] despite it has a host property', () => {
```